_health-care/_js/components/HealthCareApp.jsx

```diff
@@ -250,6 +253,18 @@ class HealthCareApp extends React.Component {
     hashHistory.push(this.getNextUrl());
   }
 
+  sectionCondition(key) {
```

Comment:
Since this is going to return either `true` or `false`, can we rename the function `isSectionBeVisible` or `shouldSectionShow` or something like that?